viadot/sources/s3.py

```diff
@@ -106,21 +106,31 @@ def cp(self, from_path: str, to_path: str, recursive: bool = False):
         """
         self.fs.copy(path1=from_path, path2=to_path, recursive=recursive)
 
-    def rm(self, path: str):
+    def rm(self, paths: list[str]):
         """
         Deletes files in a path.
 
         Args:
-            path (str): Path to a file or folder to be removed. If the path refers to
-                a folder, it will be removed recursively.
+            path (list[str]): Path to a file or folder to be removed. If the path refers to
```

Comment:
paths*. Also remember to fix the description since it used to be for the `path` param, and it's now for `paths`

Reply:
Updated in:

- https://github.com/dyvenia/viadot/pull/603/commits/f4f1643a5bae6b349718ac18542fcfea2cc342bb